play.pokemonshowdown.com/src/battle-log.ts

```diff
@@ -141,7 +142,7 @@ export class BattleLog {
 			if (battle?.ignoreOpponent) {
 				if ('\u2605\u2606'.includes(rank) && toUserid(name) !== app.user.get('userid')) return;
 			}
-			if (window.app?.ignore?.[toUserid(name)] && ' +\u2605\u2606'.includes(rank)) return;
+			if ((window.app?.ignore?.[toUserid(name)] || PS?.prefs?.ignore?.[toUserid(name)]) && ' +^\u2605\u2606'.includes(rank)) return;
```

Comment:
```suggestion
			const ignoreList = window.app?.ignore || window.PS?.prefs?.ignore;
			if (ignoreList?.[toUserid(name)] && ' +^\u2605\u2606'.includes(rank)) return;
```

It needs to be `window.PS?`, not just `PS?`. The latter will still crash if it's not defined.

Reply:
This is the last thing left! After this fix, this'll be ready to merge!